src/base/FlysystemVolume.php

```diff
@@ -343,4 +367,31 @@ protected function visibility(): string
     {
         return $this->hasUrls ? AdapterInterface::VISIBILITY_PUBLIC : AdapterInterface::VISIBILITY_PRIVATE;
     }
+
+    /**
+     * Fetch the file metadata from the volume, optionally caching the result.
+     *
+     * @param string $uri
+     * @param false $bypassCache
+     * @return array|false|mixed
```

Comment:
`mixed` can probably be removed here? `getMetadata()` says it will only return an array or `false`.